### ISSUE TITLE
Problem with detecting CLOC

### ISSUE THREAD

**Issue by dennisdegreef** [sebastianbergmann/phploc#126]
Hi,

I think I've found a bug regarding the CLOC, provided are a few examples:
PS. I'm running `phploc 2.1.4` on `php 5.6`

```php
cat loc1.php; phploc loc1.php | grep 'LOC'
<?php

/* this is a for-loop */
for ($i = 0; $i < 100; $i++)
{
    printf("hello");
}
  Lines of Code (LOC)                                7
  Comment Lines of Code (CLOC)                       0 (0.00%)
  Non-Comment Lines of Code (NCLOC)                  7 (100.00%)
  Logical Lines of Code (LLOC)                       3 (42.86%)
```

As you can see, this piece of code contains 1 comment line, while it isn't being counted as one.

```php
cat loc1.php; phploc loc1.php | grep 'LOC'
<?php

/** this is a for-loop */
for ($i = 0; $i < 100; $i++)
{
    printf("hello");
}
  Lines of Code (LOC)                                7
  Comment Lines of Code (CLOC)                       0 (0.00%)
  Non-Comment Lines of Code (NCLOC)                  7 (100.00%)
  Logical Lines of Code (LLOC)                       3 (42.86%)
  Average Complexity per LLOC                     0.33
```

Same counts for comments which are of another type internally (starting with two asteriskes)

```php
cat loc1.php; phploc loc1.php | grep 'LOC'
<?php

// this is a for-loop
for ($i = 0; $i < 100; $i++)
{
    printf("hello");
}
  Lines of Code (LOC)                                7
  Comment Lines of Code (CLOC)                       1 (14.29%)
  Non-Comment Lines of Code (NCLOC)                  6 (85.71%)
  Logical Lines of Code (LLOC)                       3 (42.86%)
```

When I change it to two forward slashes, the line gets recognised.

```php
cat loc1.php; phploc loc1.php | grep 'LOC'
<?php

/*
this is a for-loop */
for ($i = 0; $i < 100; $i++)
{
    printf("hello");
}
  Lines of Code (LOC)                                8
  Comment Lines of Code (CLOC)                       1 (12.50%)
  Non-Comment Lines of Code (NCLOC)                  7 (87.50%)
  Logical Lines of Code (LLOC)                       3 (37.50%)
```

When moving it back to the original, but putting in a newline after the starting tag, it recognises it as a single line of comment.

```php
cat loc1.php; phploc loc1.php | grep 'LOC'
<?php

/*
 *
this is a for-loop */
for ($i = 0; $i < 100; $i++)
{
    printf("hello");
}
  Lines of Code (LOC)                                9
  Comment Lines of Code (CLOC)                       3 (33.33%)
  Non-Comment Lines of Code (NCLOC)                  6 (66.67%)
  Logical Lines of Code (LLOC)                       3 (33.33%)
  Average Complexity per LLOC                     0.33
```

When I put in another line 'of comment', it suddenly also recognises not only the second line, but the third line (with text and closing tag) is now also taken into account.

I think this is a bug, and I will look if I can find it and make a pull request out of it. If I can't find it (quickly), at least there is a bugreport for it.

**Comment by dennisdegreef** [comment]
It seems the functionality is broken by https://github.com/sebastianbergmann/phploc/commit/c26bf123477d28841c3ef4d602e73bfc43ccf367

![screen shot 2015-08-09 at 01 32 09](https://cloud.githubusercontent.com/assets/361905/9152780/94bd8a86-3e36-11e5-9e18-b36caab5ee7e.png)

**Comment by dennisdegreef** [comment]
This pull request should fix this issue, and also fix #102
